tests/utils.js

```diff
@@ -205,7 +219,12 @@ describe('Utils', () => {
                 afterRefresh: sandbox.stub(),
             };
 
-            sandbox.stub(utils, 'refresh').returns(Promise.resolve({body: {access_token: 'Token-2'}}));
+            sandbox.stub(utils, 'refresh').returns(Promise.resolve({
+                body: {access_token: 'Token-2'},
+                response: {
+                    statusCode: 200,
```

Comment:
This change prevents an exception [here](https://github.com/sugarcrm/thorn/pull/210/files#diff-14ef994fa167d355bdaf9b44add54b87R150).